assets/js/components/grid-content-control/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import PropTypes from 'prop-types';
+import { ToggleControl } from '@wordpress/components';
+
+/**
+ * A combination of range controls for product grid layout settings.
+ */
+const GridContentControl = ( { onChange, settings } ) => {
+	const { button, price, title } = settings;
+	return (
+		<Fragment>
+			<ToggleControl
+				label={ __( 'Product title', 'woo-gutenberg-products-block' ) }
+				help={ title ? 'Product title is visible.' : 'Product title is hidden.' }
```

Comment:
Should the `help` attributes be passed through `__()` too?

Reply:
Good catch, yep :)